packages/modal/src/Modal.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import IconClose from 'mdi-react/CloseIcon';
+import Blanket from '@xcritical/blanket';
+
+import {
+  ModalHeaderWrapper,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalIconClose,
+} from './styles';
+
+import { IModalProps } from './interfaces';
+import { getModalBlanketTheme } from './utils';
+
+
+export const PureModal: React.FC<IModalProps> = ({
+  isOpen,
+  title,
+  children,
+  theme,
+  onModalCancel,
+}) => {
+  if (!isOpen) {
+    return null;
+  }
+
+  const modalBlanketTheme = getModalBlanketTheme(theme);
```

Comment:
Maybe this code should be optimized (for example, using 'useRef' and 'useEffect'), because in this case you get 'modalBlanketTheme' every rendering.

Reply:
Added memoize wrapper inside utils.